main.js

```diff
@@ -66,17 +87,55 @@ function setAPPListeners () {
  * Opens new window with index.html(Jitsi Meet is loaded in iframe there).
  */
 function createJitsiMeetWindow () {
-    jitsiMeetWindow = new BrowserWindow(jitsiMeetWindowOptions);
-    jitsiMeetWindow.loadURL(indexURL);
+  jitsiMeetWindow = new BrowserWindow(jitsiMeetWindowOptions);
+  jitsiMeetWindow.loadURL(indexURL);
+  microWindow = new BrowserWindow(microWindowOptions);
```

Comment:
ok, so is this just always loading the micro window for now for testing? (as opposed to only loading it when the main window loses focus?)